WISE-Community/WISE
# Styled student and teacher edit profile pages and added unit tests

To test:
- Log in as teacher and navigate to `/teacher/edit/profile`.
- Make sure changing and saving details in the Account Info form works.
- Make sure changing password works.
- Log in as student and navigate to `/student/edit/profile`.
- Test the Account Info and Password forms to make sure they work too.

Closes #1458.
